os_stub/spdm_device_secret_lib_sample/lib.c

```diff
@@ -337,6 +337,11 @@ bool libspdm_gen_csr(uint32_t base_hash_algo, uint32_t base_asym_algo, bool *nee
                                   csr_len, csr_pointer);
     libspdm_asym_free(base_asym_algo, context);
     free(prikey);
+
+    if (csr_buffer_size < *csr_len) {
+        LIBSPDM_DEBUG((LIBSPDM_DEBUG_INFO,"csr buffer is small to sotre generated csr! \n"));
```

Comment:
"csr buffer is too small to store generated csr.\n"

Reply:
Thanks. I have changed the code.